ml/Config.cc

```diff
@@ -47,6 +47,11 @@ void Config::readMLConfig(void) {
     double ADWindowRateThreshold = config_get_float(ConfigSectionML, "window minimum anomaly rate", 0.25);
     double ADDimensionRateThreshold = config_get_float(ConfigSectionML, "anomaly event min dimension rate threshold", 0.05);
 
+    double SaveAnomalyPercentageEvery = config_get_number(ConfigSectionML, "window size of anomaly bit counting for anomaly percentage", 15 * 60);
+
+    double MaxAnomalyRateInfoTableSize = config_get_number(ConfigSectionML, "the maximum size, in rows, of the data table that holds anomaly rate information", 10000.0);
```

Comment:
Maybe `MaxAnomalyRateInfoTableRows` is a more appropriate name.

Reply:
Done.